book/09-git-and-other-scms/sections/import-bzr.asc

```diff
@@ -0,0 +1,154 @@
+==== Bazaar
+(((Bazaar)))(((Importing, from Bazaar)))
+
+Bazaar is a distributed source management content like Git.
+It is very easy to convert a Bazaar repository into a Git repository thanks to Bazaar's plugin system.
+To convert a Bazaar repository into a Git repository, you have first to import the bzr-fastimport plugin.
+Here is the whole procedure if you have not yet imported any plugin:
+[source,console]
+----
+$ mkdir --parents ~/.bazaar/plugins/bzr # creates the necessary folders for the plugins
+$ cd ~/.bazaar/plugins/bzr
+$ bzr branch lp:bzr-fastimport fastimport # imports the fastimport plugin
+$ cd fastimport
+$ sudo python setup.py install --record=files.txt
+----
+
+There you go!
+You've just installed the fastimport plugin.
```

Comment:
I think it is better to add a sentence to introduce it, because I think it is important to mention that at this point, the plugin is installed. Do you agree ?